data/wicked/bonding/ifcfg-eth0

```diff
@@ -0,0 +1,2 @@
+STARTMODE='hotplug'
```

Comment:
maybe we can use just https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/wicked/ifcfg/macvtap_eth ? no point to duplicate files in data folder 
we can rename it to something not macvtap specific to not confuse

Reply:
ok, since it involves more tests, I will do it in another PR